### ISSUE TITLE
mAP result from pretrained model

### ISSUE THREAD

**Issue by dlkphuong** [deeplearning-wisc/vos#54]
Hi, I'm trying to replicate evaluation of VOS on COCO (training dataset is PASCAL-VOC) using `model_final_vos_regnet_voc,pth` according to `README.md` but I couldn't find the mAP result on the terminal. However, I do find a `mAP_res.txt` file in the output dir, but the result is different from the paper (I got 51.2 while it's 51.6±0.1). I would like to know how do you replicate mAP result in the paper? Was it with the same pretrained model?

**Comment by d12306** [comment]
Hi, @dlkphuong , thanks for checking the results. My pretrained model is with the random seed of 0. To replicate the results, you can train the regnet model with seeds 1, and 2. and then average to get the mAP. 51.2 is fairly close to 51.6 as reported.
